cmd/device/add/add.go

```diff
@@ -35,18 +37,18 @@ type DeviceFile struct {
 }
 
 func NewCommand() *cobra.Command {
+	var file string
 	cmd := &cobra.Command{
 		Use:   "add",
 		Short: "Add devices",
 		Long:  `Create the devices described in the given TOML files.`,
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("Add Devices:")
-			for _, val := range args {
-				fmt.Println(val, "... ")
-				processFile(val)
-			}
+			fname := cmd.Flag("file").Value.String()
+			processFile(fname)
```

Comment:
feels like we are processing only one toml file?

for the future -- something to consider -- size of a toml file for some kind of denial of service attack?

Reply:
yes, with adding --file flag we could process one file , that contains multiple devices

Reply:
agree, we could add enhancement issue for the size of the file